packages/client/wallets/smart-wallet/package.json

```diff
@@ -23,8 +23,10 @@
         "build-no-minify": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --dts --sourcemap",
         "dev": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --dts --sourcemap --watch",
         "link:common-sdk-base": "yalc add @crossmint/common-sdk-base && yalc link @crossmint/common-sdk-base && pnpm i && pnpm build && yalc push",
+        "link:starter": "pnpm link --global @crossmint/client-sdk-smart-wallet && pnpm --filter @crossmint/client-sdk-smart-wallet-react-starter exec pnpm link --global @crossmint/client-sdk-smart-wallet",
```

Comment:
ty, much needed!

Reply:
Might be worth spending 5 mins investigating if you should use yalc here

Reply:
np :smile: